keyboards/sofle/keymaps/via/config.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Could add a license header here, please?

Here's a sample:

https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
I added the information to all files. Very sorry I did not have that right away, the default map does not and as I did so little modification to it, I was unsure if it was my place to put a license (and my name) on it. I hope my solution is okay. Thanks for your work!